### PR TITLE
chore: add integ test for create hasMany relationship

### PR DESCRIPTION
**Pull request by bombguy**
*Issue #, if available:*

*Description of changes:*

- add test for hasMany relationship create flow

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/test-generator/integration-test-templates/src/CreateFormTests.tsx [hunk 1]
```diff
@@ -94,10 +98,12 @@ export default function CreateFormTests() {
           onSuccess={async () => {
             const records = await DataStore.query(AllSupportedFormFields);
             const record = records[0];
+
             setDataStoreFormCreateAllSupportedFormFieldsRecord(
               JSON.stringify({
                 ...record,
                 HasOneUser: await record.HasOneUser,
+                HasManyStudents: record.HasManyStudents,
```
Comment (zchenwei) [inline packages/test-generator/integration-test-templates/src/CreateFormTests.tsx:106]: I think we need to test the new lazy loading way

File: packages/test-generator/lib/models/schema.ts [hunk 2]
```diff
@@ -779,7 +779,7 @@ export default {
           isArrayNullable: true,
           association: {
             connectionType: 'HAS_MANY',
-            associatedWith: 'allSupportedFormFieldsID',
```
[the remark this answers is not in the document]
Comment (bombguy) [reply packages/test-generator/lib/models/schema.ts]: fixed 👍

**Review by zchenwei — COMMENTED** [review]
LGTM..But it seems the style test failed

**Review by zchenwei — DISMISSED** [review]
LGTM

**Review by lyonsbp — DISMISSED** [review]

**Review by zchenwei — DISMISSED** [review]
Your local snapshot is old.

**Review by hein-j — APPROVED** [review]

**Review by zchenwei — APPROVED** [review]

**Pull request merged**